### DISCUSSION TITLE
I am getting error while running dbServer.js file

### DISCUSSION THREAD

**Discussion by MrunalKashid02** [Harshdev098/Research-Nexas#68] [Q&A] [answered]
![PR2](https://github.com/user-attachments/assets/8a9f8a4b-d8c1-4eb7-8498-22f0766d44f8)

I tried serveral times by google search but i could not able to solve problem

**Comment by Harshdev098** [comment]
Try these steps- 
- Update mysql to mysql2 `npm update mysql2`
- Check whether all the parameters in the .env file is correct.
- Check whether you have setted a password for the mysql or not and ensure that mysql service is running

Let me know if this resolved this query or not

**Reply by MrunalKashid02** [reply]
I have updated mysql but getting same error. In env file what will be the value of Acess token is same or anything else.

**Reply by Harshdev098** [reply]
It will work with that also

**Reply by MrunalKashid02** [reply]
but its not working

**Reply by Harshdev098** [reply]
Does your fork is synced with this repo and also check whether your clone is up to date

**Reply by Harshdev098** [reply]
Because one of the contributor is also facing same problem he made changes on behalf of this problem(it's merged yesterday only)

**Reply by MrunalKashid02** [reply]
Yeah I synced the fork and updated the clone but still same error

**Reply by Harshdev098** [reply]
Can you send your .env file and the database grant screenshots

![Screenshot from 2024-10-08 18-21-55](https://github.com/user-attachments/assets/4c5e9cf6-1523-4ce4-806d-6480b792af8d)

**Comment by MrunalKashid02** [comment]
![Screenshot 2024-10-08 184725](https://github.com/user-attachments/assets/4a111545-f071-4447-adb4-4d6bd6a7b58c)
![Screenshot 2024-10-08 185421](https://github.com/user-attachments/assets/f480d665-cb33-4b84-975e-d06e191f7c7a)

**Reply by MrunalKashid02** [reply]
Ig I havent made mistake in .env file value

**Reply by Harshdev098** [reply, marked as answer]
Yah, actually the .env folder should in the the login-system folder, I think the server is not accessing the values from .env

**Reply by Harshdev098** [reply]
if it solves the query please update me!

**Reply by MrunalKashid02** [reply]
Thanks issue solved.

**Reply by Harshdev098** [reply]
Please mark it as a solution, it will be helpful to others

**Reply by MrunalKashid02** [reply]
How ?

**Reply by Harshdev098** [reply]
Is it not showing `mark it as answer`, if not left it
